doc/changelog/07-vernac-commands-and-options/15056-collection-variable-clash-deprecation-notice.rst

```diff
@@ -0,0 +1,5 @@
+- **Deprecated:**
+  Deprecate ambiguous :cmd:`Proof using` and :cmd:`Collection` usage
```

Comment:
Bit late for this now, but perhaps we didn't need `Deprecate` here since it already said `**Deprecated:**`. @Zimmi48 Do you think this is worth changing?

Reply:
Indeed, we usually don't repeat the verb, although this is a common mistake (that I generally notice and suggest fixing while reviewing the changelog consolidation PRs). Maybe we should edit the changelog template once more to insist on this caveat.